webpack/assets/javascripts/react_app/components/PasswordStrength/__tests__/__snapshots__/PasswordStrength.test.js.snap

```diff
@@ -3,8 +3,11 @@
 exports[`PasswordStrength component rendering renders password-strength 1`] = `
 <div>
   <CommonForm
+    className=""
```

Comment:
should it be empty?

Reply:
Yes!
The prop `className` of `CommonForm` set to be an empty string by default with default-props.